src/figuresClasses.ts

```diff
@@ -1,19 +1,68 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || c >= a + b) {
+      throw new Error('your error message');
+    }
+  }
 
-}
+  shape = 'triangle';
 
-export class Circle {
+  getArea(): number {
+    const p = 0.5 * (this.a + this.b + this.c);
 
+    const square = Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c));
+
+    return parseFloat(square.toFixed(3).slice(0, -1));
+  }
 }
 
-export class Rectangle {
+export class Circle implements Figure {
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('your error message');
```

Comment:
change the text ``your error message'' to something more meaningful. Fix it everywhere

Reply:
Done